### ISSUE TITLE
CSV and Excel report String type for a column with numbers which fools the comparator

### ISSUE THREAD

**Issue by tomaszguzialek** [datacleaner/DataCleaner#432]
We fixed issue #360 by using a different comparator for numeric columns, different for dates and different for everything else (Strings). It is determined which comparator to use based on the column type associated with the column we want to sort on. It works well for databases as they report numeric type, but CSV files and Excel spreadsheet always report String type and the values are always sorted lexicographically.

**Comment by kaspersorensen** [comment]
Hmm wasn't this exactly what #360 solved?? I mean, with databases (Insert into table component) we do not even have a sort function.

**Comment by tomaszguzialek** [comment]
The problem is a little different. Consider two job: 
Database as a source --> Create CSV file analyzer with sorting
CSV file as a source --> Create CSV file analyzer with sorting

The first one gives us information that the column we want to sort on is a numeric column - we can sort it numerically. In the second job, our numeric column is actually of type String - the values will be sorted lexicographically.

However, I have just tested that adding a ConvertToNumberTransformer solves the issue (this was not giving any effect before #360 was solved). Therefore, I will close this issue as kinda irrelevant.
